types/codemirror/index.d.ts

```diff
@@ -1256,6 +1256,13 @@ declare namespace CodeMirror {
 
         /** Optional lint configuration to be used in conjunction with CodeMirror's linter addon. */
         lint?: boolean | LintStateOptions | Linter | AsyncLinter;
+
+        /** Allows you to configure the behavior of mouse selection and dragging. The function is called when the left mouse button is pressed. */
+        configureMouse?: (
```

Comment:
Could this moved between `extraKeys` and `lineWrapping` to match the position in the manual documentation?

Reply:
Done!